### ISSUE TITLE
这种输出是不是表明训练成功了，后续只需要对模型进行调整就可以了。（golang的菜鸟辛苦您了）公司还没python 所以我来了

### ISSUE THREAD

**Issue by Mr1994** [ymcui/Chinese-LLaMA-Alpaca-2#435]
### 提交前必须检查以下项目

- [X] 请确保使用的是仓库最新代码（git pull），一些问题已被解决和修复。
- [X] 我已阅读[项目文档](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki)和[FAQ章节](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki/常见问题)并且已在Issue中对问题进行了搜索，没有找到相似问题和解决方案。
- [X] 第三方插件问题：例如[llama.cpp](https://github.com/ggerganov/llama.cpp)、[LangChain](https://github.com/hwchase17/langchain)、[text-generation-webui](https://github.com/oobabooga/text-generation-webui)等，同时建议到对应的项目中查找解决方案。

### 问题类型

模型训练与精调

### 基础模型

Chinese-LLaMA-2 (7B/13B)

### 操作系统

Linux

### 详细描述问题

```
{'loss': 1.0194, 'learning_rate': 1.2055455093429777e-07, 'epoch': 0.0}
{'loss': 0.9914, 'learning_rate': 1.2055455093429778e-06, 'epoch': 0.0}
{'loss': 0.9185, 'learning_rate': 2.4110910186859556e-06, 'epoch': 0.0}
{'loss': 0.8991, 'learning_rate': 3.6166365280289336e-06, 'epoch': 0.0}
{'loss': 0.8975, 'learning_rate': 4.822182037371911e-06, 'epoch': 0.0}
{'loss': 0.87, 'learning_rate': 6.027727546714889e-06, 'epoch': 0.0}
{'loss': 0.8098, 'learning_rate': 7.233273056057867e-06, 'epoch': 0.0}
  0%|▌                                                                                                                                                                                                                                                                                                                              | 61/33161 [04:08<37:11:24,  4.04s/it]
{'loss': 0.78, 'learning_rate': 8.438818565400844e-06, 'epoch': 0.0}
{'loss': 0.7271, 'learning_rate': 9.644364074743822e-06, 'epoch': 0.0}
{'loss': 0.7858, 'learning_rate': 1.0849909584086799e-05, 'epoch': 0.0}
{'loss': 0.7786, 'learning_rate': 1.2055455093429777e-05, 'epoch': 0.0}
{'loss': 0.7859, 'learning_rate': 1.3261000602772756e-05, 'epoch': 0.0}
{'loss': 0.7048, 'learning_rate': 1.4466546112115734e-05, 'epoch': 0.0}
{'loss': 0.7505, 'learning_rate': 1.5672091621458713e-05, 'epoch': 0.0}
{'loss': 0.736, 'learning_rate': 1.6877637130801688e-05, 'epoch': 0.0}
{'loss': 0.7211, 'learning_rate': 1.8083182640144666e-05, 'epoch': 0.0}
{'loss': 0.7201, 'learning_rate': 1.9288728149487645e-05, 'epoch': 0.0}
{'loss': 0.7189, 'learning_rate': 2.049427365883062e-05, 'epoch': 0.01}
  1%|█▋                                                                                                                                                                                                                                                                                                                            | 173/33161 [11:41<37:04:42,  4.05s/it]```


### 依赖情况（代码类问题务必提供）

```
# 请在此处粘贴依赖情况（请粘贴在本代码块里）
```
![Uploading 1701657190966.jpg…]()


### 运行日志或截图

```
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/15978040/2264e0c6-9f11-430e-94c3-a7f6b96689d0)```

**Comment by q5756578** [comment]
可以观察评价集的loss eval loss  是否正常
